spec/shr_questionnaire_map_stu3.txt

```diff
@@ -0,0 +1,120 @@
+Grammar:	Map 5.1
+Namespace:	shr.core
+Target:		FHIR_STU_3
+
+
+Questionnaire maps to Questionnaire:
```

Comment:
Even though Questionnaire maps to STU3 Questionnaire, the generated FHIR IG shows several mapped elements in the Differential table.